addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -93,6 +93,13 @@
 		<state pattern="%.2f A" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="current" advanced="true">
```

Comment:
What is the difference between the existing _instantamp_ and this new _current_?